guides/operator/custom-boms.md

```diff
@@ -170,6 +170,9 @@ As a result, a BOM at version `${VERSION}` can be found at
 subcomponent `${SUBCOMPONENT}` at version `${SUBCOMPONENT_VERSION}` can be
 found at `gs://${BUCKET}/${SUBCOMPONENT}/${SUBCOMPONENT_VERSION}/${PROFILE}`.
 
+If Halyard is unable to read from the GCS bucket, please see the troubleshooting
+instructions [here](https://www.spinnaker.io/setup/quickstart/faq/#halyard-times-out-during-a-config-change).
```

Comment:
Instead of "...troubleshooting instructions [here]().", maybe just "...[troubleshooting instructions]()."